src/Common/Core/Impl/Telemetry/TelemetryServiceBase.cs

```diff
@@ -32,13 +32,13 @@ public abstract class TelemetryServiceBase : ITelemetryService, IDisposable {
         /// </summary>
         public bool IsEnabled {
             get {
-                return this.TelemetryRecorder.IsEnabled;
+                return this.TelemetryRecorder?.IsEnabled == true;
```

Comment:
I'm not entirely sure about the cause, but I was hitting this in the tests sporadically. Doesn't seem to be related to the changes.